dev/test/bulk-writer.ts

```diff
@@ -125,31 +125,40 @@ describe('BulkWriter', () => {
     const writeResults = responses.map(response => ({
```

Comment:
Please use proper types for `successResponse`, `failResponse` and `createResponse` (something like ` writeResult: WriteResult, status: Status }`. You could even change successResponse to return a single valid api.iBatchWriteResponse, which you could then use in all cases where only one response is needed. You can do the same for `failResponse`.  `createResponse` would just merge results.

See diff with proper types: https://gist.github.com/schmidt-sebastian/292dcc2bfdc2ad3309e480ed57251a1e